### PR TITLE
fix: parseExportNamesInto specifiers typo

### PR DESCRIPTION
**Pull request by btea**
fix typo :  

specificers  -> specifiers

### CODE REVIEW

File: packages/react-server-dom-webpack/src/ReactFlightWebpackNodeLoader.js [hunk 1]
```diff
@@ -210,10 +210,10 @@ async function parseExportNamesInto(
...
           }
         }
-        if (node.specificers) {
-          const specificers = node.specificers;
-          for (let j = 0; j < specificers.length; j++) {
-            addExportNames(names, specificers[j].exported);
+        if (node.specifiers) {
+          const specifiers = node.specifiers;
+          for (let j = 0; j < specifiers.length; j++) {
+            addExportNames(names, specifiers[j].exported);
```
Comment (bvaughn) [inline packages/react-server-dom-webpack/src/ReactFlightWebpackNodeLoader.js:216]: How did this ever work?

Comment (btea) [reply packages/react-server-dom-webpack/src/ReactFlightWebpackNodeLoader.js:216]: Sorry, I didn't understand your question.

Comment (bvaughn) [reply packages/react-server-dom-webpack/src/ReactFlightWebpackNodeLoader.js:216]: How did the code that was there before– `node.specificers` –ever work?

Comment (btea) [reply packages/react-server-dom-webpack/src/ReactFlightWebpackNodeLoader.js:216]: I'm confused, too.

Comment (gaearon) [reply packages/react-server-dom-webpack/src/ReactFlightWebpackNodeLoader.js:216]: It probably didn’t. :) would be good to recreate a case where this codepath runs.

Comment (bvaughn) [reply packages/react-server-dom-webpack/src/ReactFlightWebpackNodeLoader.js:216]: I tagged Sebastian on the original PR.

**Review by bvaughn — APPROVED** [review]

**Pull request merged**